op-node/rollup/derive/attributes.go

```diff
@@ -194,8 +194,8 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 		Withdrawals:           withdrawals,
 		ParentBeaconBlockRoot: parentBeaconRoot,
 	}
-	if ba.rollupCfg.IsHolocene(nextL2Time) {
-		r.EIP1559Params = new(eth.Bytes8)
+	if ba.rollupCfg.IsJovian(nextL2Time) || ba.rollupCfg.IsHolocene(nextL2Time) {
```

Comment:
Since Holocene is always active when Holocene is active by definition, `IsJovian || IsHolocene` is always equivalent to `IsHolocene`.